src/api/bulkQuotes.js

```diff
@@ -58,24 +59,34 @@ module.exports = {
       'Publish HTTP POST bulkQuotes request',
       ['success']
     ).startTimer()
+    const errorCounter = Metrics.getCounter('errorCount')
+    let step
 
     try {
       await util.auditSpan(request)
 
       const { topic, config } = kafkaConfig.PRODUCER.BULK_QUOTE.POST
       const topicConfig = dto.topicConfigDto({ topicName: topic })
+      step = 'messageFromRequestDto-1'
       const message = await dto.messageFromRequestDto({
         request,
         type: Events.Event.Type.BULK_QUOTE,
         action: Events.Event.Action.POST
       })
-
+      step = 'produceMessage-2'
       await Producer.produceMessage(message, topicConfig, config)
 
       histTimerEnd({ success: true })
       return h.response().code(Http.ReturnCodes.ACCEPTED.CODE)
     } catch (err) {
       histTimerEnd({ success: false })
+      const fspiopError = reformatFSPIOPError(err)
+      errorCounter.inc({
+        code: fspiopError?.apiErrorCode.code,
+        system: undefined,
+        operation: 'postBulkQuotes',
+        step
+      })
```

Comment:
I am approving the PR, but please consider if you want to avoid this repetitive code by maybe moving it to `util.rethrowFspiopError(err)`, so that it can be called like this:
```js
util.rethrowFspiopError(err, operation, step)
```